src/modules/cron/cron.controller.ts

```diff
@@ -0,0 +1,60 @@
+import { Controller, HttpCode, HttpStatus, Put } from '@nestjs/common';
+import {
+    ApiCreatedResponse,
+    ApiBadRequestResponse,
+    ApiUnauthorizedResponse,
+    ApiForbiddenResponse,
+    ApiNotFoundResponse,
+    ApiInternalServerErrorResponse,
+    ApiBearerAuth,
+    ApiOAuth2,
+    ApiTags,
+} from '@nestjs/swagger';
+import { PersonRepository } from '../person/persistence/person.repository.js';
+import { KeycloakUserService } from '../keycloak-administration/domain/keycloak-user.service.js';
+import { DomainError } from '../../shared/error/domain.error.js';
+
+@Controller({ path: 'cron' })
+@ApiBearerAuth()
+@ApiOAuth2(['openid'])
+@ApiTags('cron')
+export class CronController {
+    public constructor(
+        private readonly keyCloakUserService: KeycloakUserService,
+        private readonly personRepository: PersonRepository,
+    ) {}
+
+    @Put('kopers-lock')
+    @HttpCode(HttpStatus.OK)
+    @ApiCreatedResponse({ description: 'User were successfully locked.', type: Boolean })
+    @ApiBadRequestResponse({ description: 'User are not given or not found' })
+    @ApiUnauthorizedResponse({ description: 'Not authorized to lock user.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to lock user.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to lock user.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while trying to lock user.' })
+    public async KoPersUserLock(): Promise<boolean> {
+        try {
+            const keyCloakIds: string[] = await this.personRepository.getKoPersUserLockList();
+            if (keyCloakIds.length === 0) {
+                return false;
+            }
```

Comment:
Should this be `false`? In my opinion, locking no users because there aren't any to lock, is still a success.